contacts.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+<title>Contacts</title>
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+<link rel="stylesheet" type="text/css" href="static/css/schedule_style.css" media="screen" />
+<link rel="stylesheet" type="text/css" href="static/css/font.css" />
+<link rel="stylesheet" type="text/css" href="static/css/demo.css" />
+<link rel="stylesheet" type="text/css" href="static/css/stylebg.css" />
+<link rel="stylesheet" type="text/css" href="static/css/demo_bmenu.css" />
+<link rel="stylesheet" type="text/css" href="static/css/style_bmenu.css" />
+<style type="text/css">
+<!--
```

Comment:
@polybuildr 

Please get all the assignees to remove the comments. Reduce the bandwidth.